### PR TITLE
LPS-167253 Make all things consistent

### PR DESCRIPTION
**Pull request by seltong**
https://github.com/brianchandotcom/liferay-portal/pull/133975

### CODE REVIEW

File: modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/com/liferay/frontend/data/set/views/web/batch/data-set-views.json [hunk 1]
```diff
@@ -275,9 +275,9 @@
 				{
 					"deletionType": "prevent",
 					"label": {
-						"en_US": "Creation Menu Primary Actions"
+						"en_US": "FDS Creation Menu Primary Actions"
```
Comment (gabrielwas) [inline modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/com/liferay/frontend/data/set/views/web/batch/data-set-views.json:278]: Creation Menu Primary FDS Actions

File: modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/com/liferay/frontend/data/set/views/web/batch/data-set-views.json [hunk 2]
```diff
@@ -297,9 +297,9 @@
 				{
 					"deletionType": "prevent",
 					"label": {
-						"en_US": "Entry Views"
+						"en_US": "FDS Entry Views"
 					},
-					"name": "entryViews",
+					"name": "fdsEntryViews",
```
Comment (gabrielwas) [inline modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/com/liferay/frontend/data/set/views/web/batch/data-set-views.json:302]: @seltong the prefix should be in uppercase letter. `FDSEntryViews`

Apply also for the others

**Review by gabrielwas — CHANGES_REQUESTED** [review]

File: modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/com/liferay/frontend/data/set/views/web/batch/data-set-views.json (hunk 2 above)
Comment (seltong) [reply modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/com/liferay/frontend/data/set/views/web/batch/data-set-views.json:302]: I was using [this](https://github.com/liferay/liferay-portal/blob/b0f9ff167391a91351f64198cbc567acdbf1f29f/modules/apps/site-initializer/site-initializer-extender/site-initializer-extender-test-bundle-2/src/main/resources/site-initializer/object-relationships/test-object-relationship-2.json#L6) example as a base

**Comment by seltong** [comment]
ci:test:sf

**Comment by seltong** [comment]
Resent here: https://github.com/brianchandotcom/liferay-portal/pull/134292

**Pull request closed without merging**